stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/ObservableClientConnectorListener.java

```diff
@@ -64,7 +65,7 @@ public void onError(Throwable throwable) {
 
     private void addHttpStatusCode(int statusCode) {
         Optional<ObserverContext> observerContext = ObserveUtils.getObserverContextOfCurrentFrame(context.getStrand());
-        observerContext.ifPresent(ctx -> ctx.addTag(ObservabilityConstants.TAG_KEY_HTTP_STATUS_CODE,
-                String.valueOf(statusCode)));
+        observerContext.ifPresent(ctx -> ctx.addTag(ObservabilityConstants.TAG_KEY_HTTP_STATUS_CODE_GROUP,
+                String.valueOf(statusCode).charAt(0) + STATUS_CODE_GROUP_SUFFIX));
```

Comment:
Can we do an integer division here instead of `String.valueOf(statusCode).charAt(0)` ? I think it might be better.

Reply:
Fixed with f22240bfddcb1eaaa1c7c232136fd264c35b9b78